.github/workflows/update-vrt.yml

```
@@ -0,0 +1,40 @@
+        env:
+          COMMIT_SHA: ${{ steps.comment-branch.outputs.head.sha }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: ./.github/actions/netlify-wait-for-build
+      - name: Set up environment
+        uses: ./.github/actions/setup
+      - name: Run VRT Tests on Netlify URL
+        run: yarn vrt --update-snapshots
+        env:
+          E2E_START_URL: ${{ needs.netlify.outputs.netlify_url }}
+      - name: Commit and push changes
+        working-directory: ${{ inputs.working-directory }}
```

actionlint

property "working-directory" is not defined in object type {}

(expression)